### DISCUSSION TITLE
Leaderboard

### DISCUSSION THREAD

**Discussion by mzxyz** [subquery/subql#1070] [Frontier TestNet - Bug Reporting]
## Instruction for Leaderboard

-  [Missions] Check progress of challenges for the connected account
<img width="1689" alt="Screen Shot 2022-05-26 at 9 11 12 PM" src="https://user-images.githubusercontent.com/8177474/170458982-398bf9b1-662b-415a-b55d-016cefb81323.png">

- [Leaderboard] Check the leaderboard
<img width="1707" alt="Screen Shot 2022-05-26 at 9 11 41 PM" src="https://user-images.githubusercontent.com/8177474/170459018-1c5bb3ca-0918-425f-9502-82eac8cbf6e3.png">

- Search specific indexer with address on leaderboard
<img width="1701" alt="Screen Shot 2022-05-26 at 9 12 26 PM" src="https://user-images.githubusercontent.com/8177474/170459185-2b315650-d5b5-48a0-8c76-558cdf70fb8c.png">

- Press the blue points to jump into the challenges detail page of an indexer
<img width="1688" alt="Screen Shot 2022-05-26 at 9 12 44 PM" src="https://user-images.githubusercontent.com/8177474/170459513-5632a213-5559-4a78-8143-0b1d952be703.png">

**Comment by mzxyz** [comment]
Feel free to leave you questions and ideas.

**Comment by vvayffaring** [comment]
![Screenshot 2022-05-26 at 13 17 01](https://user-images.githubusercontent.com/36548646/170468634-281eaefd-43bc-4420-a66b-c95139f361ca.png)

![Screenshot 2022-06-06 at 14 41 00](https://user-images.githubusercontent.com/36548646/172561106-2f7293be-7250-44c6-935c-e42d0eae0e4c.png)


1. For both projects annouce not indexing
2. for both projects "remove project"
3. Go to accounts and "Unregister"
complete all steps, but can't re-register

**Comment by S3VENK** [comment]
![image](https://user-images.githubusercontent.com/94055429/170472786-cdbb9c1f-e775-4d3a-add0-9cf4ae18f128.png)
Could you kindly explain how GATEWAY-ACTIVE is calculated on a daily basis? I see the top rank get a score every day, but I only have one day. Is it related to the VM's authorization port? Because I'm using a Google cloud VM with IP 8000 set to 0.0.0.0/0:8000 because MYIP:8000 isn't working on the firewall, I'm deleting IP:8000 after the run. 

PS. I ran Frontier Season 2 with friends who used an AWS VM; I noticed they were able to complete the project 100 percent on Polkadot, but my network was not as comprehensive as theirs. Thank you, and I promise to do better next time.

**Comment by kw1knode** [comment]
The post to the unregister challenge was done on 5/19 which I completed after posting. However 100 availability points have been awarded to others for the 20th.

**Comment by Evgeny8881** [comment]
Why i can't see mission field on the top?
<img width="1137" alt="Снимок экрана 2022-05-27 в 18 18 20" src="https://user-images.githubusercontent.com/78561873/170738906-d9156317-f639-4637-a0e5-ebca82b177b4.png">

**Reply by vvayffaring** [reply]
Most likely you have turned off the server, our team is also working on the front-end, you can use the direct link 
[https://frontier.subquery.network/missions](https://frontier.subquery.network/missions)

**Reply by Evgeny8881** [reply]
Thank you

**Comment by Gribnik2K** [comment]
Hello. I followed this [instruction ](https://github.com/subquery/subql/discussions/1044)in full, met the deadline. Made a transaction, received the status "NOT INDEXING"
But it didn't count in the final count. 
My accont is 0x45ba0C72C232c196Ee3a857127B9F21Cd5147036
Pls check it )

**Reply by mzxyz** [reply]
Check your account and it was unregistered. Could you confirm that the `unregister` transaction was sent before the end of the season2: `Mon May 23 2022 11:59:00 GMT+1200`, thanks.

**Reply by Gribnik2K** [reply]
Hello. Yes, I think I understand - my mistake is that I considered the [instructions ]( https://github.com/subquery/subql/discussions/1044)I gave above necessary and sufficient to complete this quiz. Later on May 22, I already found how to do it right. It is a pity that it was impossible to immediately check the completion immediately, as with other quizzes. The [instruction](https://github.com/subquery/subql/discussions/1044)  misled me a little. Of course it's not entirely obvious, but okay. I hope to complete all tasks in the next series of quizzes. After all, there is no mention of this in the instructions:
![image](https://user-images.githubusercontent.com/42679698/172557897-41e3e375-6aa6-4640-9307-b527a6adea99.png)
I sent exactly UNREGISTR later on May 22.
And other actions were carried out BEFORE May 22.

All in all.
In addition to the [instructions](https://github.com/subquery/subql/discussions/1044), it was necessary to additionally do what is in the screenshot.

**Comment by TommyAngeIo** [comment]
Hello dear team of the SubQuery project !

The task with **UNREGISTER_INDEXER** was completed according to the [instructions](https://github.com/subquery/subql/discussions/1044), completely and within the time frame you announced. Unfortunately, in the "**Missions**" tab, I do not have this task as completed, as well as the points accrued for it are missing. 
Perhaps there was some kind of error with the calculation. Please sort this out. Thanks

Tommy Indexer
0xeeeD5Dd49C1dFf8275693199F7F047F68512fD55

![Без имени](https://user-images.githubusercontent.com/103051909/171620898-a05c78a3-31ae-43e1-893d-d96d513f1974.png)

**Reply by mzxyz** [reply]
Check your account and it was unregistered. Could you confirm that the `unregister` transaction was sent before the end of the season2: `Mon May 23 2022 11:59:00 GMT+1200`, thanks.

**Reply by TommyAngeIo** [reply]
What is indicated in the [instructions](https://github.com/subquery/subql/discussions/1044) turned out to be insufficient. 
And all we had to do was essentially delete the indexer's account. I did it, but much later because of a misunderstanding of how to do it correctly, although the task itself is in fact very simple.Thanks.

**Comment by nikgrego** [comment]
Can you please check my points as well? I've followed the [instruction](https://github.com/subquery/subql/discussions/1044) according to deadlines, but I have only 330 points and "Incomplete" tag for UNREGISTER_INDEXER in the missions tab. Thanks in advance! 🙏

0xD0d6F8F916979691f3121C742fDe17653c0c0B3e

**Reply by mzxyz** [reply]
Hi @nikgrego your account is still registered, could you follow this https://github.com/subquery/subql/discussions/1044#discussioncomment-2787191 instruction to make sure all the historical projects are unindexing, thanks

**Reply by nikgrego** [reply]
The strange thing is I already did it before the end of the season2 and all of the projects were in the status of NOT INDEXING including new ones and as well as old: 
- QmTLPAeVhNNxU69NhcsKHRrCXBhmQYykqkszwETRS75A76
- QmTQTnBTcvv3Eb3M6neDiwuubWVDAoqyAgKmXtTtJKAHoH
- QmVcksANdHhnA2YEguJ8LvsftNkipTwkgV9eJyDPZFUqHm

I checked it twice, cause as I see in comments - people said no projects should be INDEXING.

**Reply by nikgrego** [reply]
I think I misunderstood the assignment as did Gribnik2K. I switched all projects to the status NOT INDEXING, but did not UNREGISTER at the account page. The [instruction](https://github.com/subquery/subql/discussions/1044) misled me a little too :( I will read more careful next time. Thanks for checking.

**Comment by vostok3** [comment]
Lilia#3086
vostok3vosgtok@gmail.com
0x947121cB21B2466060f53d20B265100D912e652d

same thing as https://github.com/nikgrego -> did all tasks as well as the unregister task, but only getting 330 points due to the unregister task being not counted.

**Reply by mzxyz** [reply]
Check your account and it was unregistered. Could you confirm that the `unregister` transaction was sent before the end of the season2: `Mon May 23 2022 11:59:00 GMT+1200`, thanks.

**Reply by vostok3** [reply]
I think I misunderstood the assignment as did nikgrego and Gribnik2K. I also switched all projects to the status NOT INDEXING, but did not UNREGISTER at the account page. For me as well, the https://github.com/subquery/subql/discussions/1044 was misleading :(

**Comment by BrainKiller-Crypto** [comment]
w124eth#5224
Can you please check my points 0xF767351d2378ffF20632409E55CCdf38021a8a5b 
I have only 280 points ! 
I see that I have Incomplete WITHDRAW_CLAIMED and UNREGISTER_INDEXER, but I did it this! 
 followed all instruction -  https://github.com/subquery/subql/discussions/1044in 
Thanks

**Reply by mzxyz** [reply]
Hi @BrainKiller-Crypto  your account is still registered, could you follow this https://github.com/subquery/subql/discussions/1044#discussioncomment-2787191 instruction to make sure all the historical projects are unindexing, thanks

**Reply by BrainKiller-Crypto** [reply]
Thanks for feedback I could have made my mistake not to make UNREGISTER_INDEXER, but I definitely did WITHDRAW_CLAIMED

**Comment by MGavrilo** [comment]
Hi, can you please check my points?
I see that I have Incomplete WITHDRAW_CLAIMED, SERVICE_AGREEMENT and CLAIM_REWARD, but I completed them
0x6704b31FBF1CE359b61E30f7010e7963E48dF0Ad
mary75#0325
